packages/babel-preset-env/test/fixtures/corejs3/usage-in/output.mjs

```diff
@@ -0,0 +1,10 @@
+import "core-js/modules/es.array.includes";
+import "core-js/modules/es.array.iterator";
+import "core-js/modules/es.object.entries";
+import "core-js/modules/es.object.from-entries";
+import "core-js/modules/es.object.values";
+'entries' in Object;
```

Comment:
I'm not sure about this. I think that if someone uses
```js
if ("entries" in Object)
```
or
```js
if (typeof Symbol.iterator !== "undefined")
```
they don't want to load the polyfill: they are explicitly checking if that feature is supported.

Ref: https://github.com/babel/babel/pull/8947

Reply:
However, they should be loaded for preventing conflicts. As you can see in #8947, they should be resolved as `true`.